### PR TITLE
Envelop v2+animation

### PR DESCRIPTION
**Pull request by qqumn**
image

image

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -37,10 +37,14 @@ en:
     contact:
       title: Contact us
       name_placeholder: Your name
+      name_error: Please, input real name
       email_placeholder: Your email
+      email_error: Please, input valid email
       message_placeholder: Text message
+      message_error: Body have to contain at least 10 characters
```
Comment (DeeMak13) [inline config/locales/en.yml]: How about something like:
```
contact:
  placeholders:
    name: Your name
    email: Your email
    message: Text message
  errors:
    name: Please, input real name
    email: Please, input valid email
    message: Body have to contain at least 10 characters
```

**Comment by galulex** [comment]
@qqumn are you going to do the updates here?

**Comment by qqumn** [comment]
@galulex I'm waiting 4 UR approving. 
I'll provide mobile version of this envelope, but a bit latter.

We've 2 provide full-site-responsive to take part in **awwwards**.

**Comment by qqumn** [comment]
Thank U, @DeeMak13, 4 improvements! :+1: /play clowntown

**Pull request merged**